### PR TITLE
Fix kubelet upgrade on Container Linux

### PR DESCRIPTION
**Pull request by ainmosni**
**What this PR does / why we need it**:

On Container Linux, kubelet locks itself, this patch stops kubelet
before downloading the new binaries and then starts it back up once
everything is ready.

```release-note
* Fix issue where kubelet wouldn't properly upgrade on Container Linux
```

### CODE REVIEW

File: pkg/upgrader/upgrade/kubernetes_binaries.go [hunk 1]
```diff
@@ -59,8 +59,12 @@ curl -L "https://github.com/containernetworking/plugins/releases/download/{{ .CN
 
 RELEASE="v{{ .KUBERNETES_VERSION }}"
 
+sudo systemctl stop kubelet
```
Comment (kron4eg) [inline pkg/upgrader/upgrade/kubernetes_binaries.go]: I'd prefer stopping kubelet only after download already finished and we know for real it's OK disk

Comment (ainmosni) [reply pkg/upgrader/upgrade/kubernetes_binaries.go]: Done :)

File: pkg/upgrader/upgrade/kubernetes_binaries.go [hunk 2]
```diff
@@ -59,10 +59,18 @@ curl -L "https://github.com/containernetworking/plugins/releases/download/{{ .CN
 
 RELEASE="v{{ .KUBERNETES_VERSION }}"
 
-sudo mkdir -p /opt/bin
-cd /opt/bin
+
+
+sudo mkdir -p /var/tmp/kube-binaries
```
Comment (xmudrii) [inline pkg/upgrader/upgrade/kubernetes_binaries.go]: Can we maybe use `/tmp`? Does it exist on CoreOS out of box?

**Comment by kron4eg** [comment]
/hold

@xmudrii please unhold once done

File: pkg/upgrader/upgrade/kubernetes_binaries.go (hunk 2 above)
Comment (ainmosni) [reply pkg/upgrader/upgrade/kubernetes_binaries.go]: I tested it on our machines, and yes it does.

**Pull request merged**